.github/workflows/free-disk-space/action.yml

```diff
@@ -15,7 +16,9 @@ runs:
   steps:
       - name:  Free disk space
         run: |
+          sudo apt-get update
```

Comment:
As safety measure, this is ok. However, we should not specify removals that are actually translated into package replacements.

Reply:
The problem is that package dependencies might change over time and we likely will run into the situation again. If you look at the logs, still packages are installed / upgraded (e.g. `libglvnd0`). These external changes will break the CI from time to time which is bad because it is totally unrelated to the containerized build of the layer. We also do not pin the CI image (we use `ubuntu-latest`), so whenever Github releases a new image with changed dependencies, we might have the same issue again.

That's the reason I recommend to cherry-pick both changes.
Hopefully github will provide some minimal image soon.

Reply:
I rather prefer that hard warning that something is broken because packages are not only removed.